spring-data-rest/pom.xml

```diff
@@ -43,6 +52,8 @@
     </build>
 
     <properties>
+        <start-class>com.baeldung.SpringDataRestApplication</start-class>
```

Comment:
Does this break other demos in the same module?

Reply:
No it doesn't. The Spring-boot Maven plugin in parent-boot-2 module expects the start-class property as well. So added the value